reference/artifacts/in-kubernetes-v2/index.md

```diff
@@ -74,9 +74,9 @@ new version suffix (`-vNNN`). This is critical to supporting immutable
 deployments, as rolling out new ConfigMaps, secrets, or other versioned
 resources should require any manifests that reference them to be updated as
 well. Luckily, Spinnaker makes handling these updates easy as explained
-[below](#consuming-artifacts-in-manifests).
+[below](#binding-artifacts-in-manifests).
 
-# Consuming Artifacts in Manifests
+# Binding Artifacts in Manifests
```

Comment:
"Binding...in." Is it more correct to say "Binding...to"?

Reply:
I think "Binding ... in" is appropriate, since the binding happens "inside" the manifest definition. I think of it as the same as the "bindings in" usage here: https://www.cs.cmu.edu/~rwh/introsml/core/decls.htm